cmd/standalone/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:1.7
```

Comment:
there may be a lighter base image than the one with full go tool chain (add TODO)